Tests/Marketplace/configure_and_install_packs.py

```diff
@@ -178,8 +143,6 @@ def main():
 
         if options.ami_env == "XSIAM":
             xsiam_configure_and_install_flow(options, branch_name, build_number)
-        elif options.override_all_packs:
-            xsoar_configure_and_install_all_packs(options, branch_name, build_number)
```

Comment:
so far override all packs did not affect XSIAM?

Reply:
Yes